app/src/main/java/com/dsu/poc/ui/login/LoginViewModel.kt

```diff
@@ -1,4 +1,18 @@
 package com.dsu.poc.project.ui.login
 
-class LoginViewModel {
+import android.app.Application
+import androidx.lifecycle.AndroidViewModel
+import androidx.lifecycle.LiveData
+import com.dsu.poc.repository.login.ValidationRepository
+
+class LoginViewModel: AndroidViewModel {
```

Comment:
add class level java documentation and function level java documentation with parameter description